packages/fsapp/src/beta-app/development/dev-menu.component.web.tsx

```diff
@@ -70,17 +70,17 @@ const styles = StyleSheet.create({
   }
 });
 
-const extractDevRoutes = (routes?: Routes, prefix: string= ''): string[] => {
+const extractDevRoutes = (routes?: Routes): string[] => {
   return (
     routes?.reduce<string[]>(
       (prev, route) => [
         ...prev,
         ...((('component' in route || 'loadComponent' in route) && route.quickDevMenu) ||
         'children' in route
           ? [
-            `${prefix}/${route.path ?? ''}`,
+            'initialPath' in route ? `/${route.initialPath ?? ''}` : `/${route.path ?? ''}`,
             ...('children' in route
-                ? extractDevRoutes(route.children, `${prefix}/${route.path ?? ''}`)
+                ? extractDevRoutes(route.children)
```

Comment:
This looks like it will need the `prefix` still because it is using `route.children`

Reply:
Thanks, good catch.  It's now updated.